### PR TITLE
Correct misspelled words in the docs and README #Issue 596

### PR DESCRIPTION
**Pull request by antonio-pedro99**

### CODE REVIEW

File: site/infra.ngs [hunk 1]
```diff
@@ -58,5 +58,5 @@
     #            }
     #        ]
     # * Create CloudFront distribution pointing to each s3 public address, with alternate domain name (cname) `${REDIRECTS.id}.${REDIRECTS.host}`, use only North America and Europe, with redirect http to http, create ssl cert, TLSv1.2_2018, HTTP/2, no Default root object
-    # * Create DNS alias for all addressess that points to each cloudfront address
+    # * Create DNS alias for all addressees that points to each cloudfront address
```
Comment (ilyash-b) [inline site/infra.ngs:61]: Typo in fixing typo. Should be `addresses`

Comment (antonio-pedro99) [reply site/infra.ngs]: please do review it again. 

I apologize for the mistake.

Comment (ilyash-b) [reply site/infra.ngs]: > please do review it again.

There is a button for that in the UI somewhere. Try to find it for next time.

> 
> I apologize for the mistake.

Completely optional step :)
It is assumed that people make mistakes.

**Comment by antonio-pedro99** [comment]
@ilyash-b  please consider closing the i#Issue 596 as fixed

**Pull request merged**